docs/release_notes/v0.3.0.md

```diff
@@ -0,0 +1,2 @@
+
+Reverted shellcheck version to 0.7.0 to resolve the issue of additional dependencies.
```

Comment:
How about saying something like "Locked shellcheck version used in the action to 0.7.0 instead of using the latest. This gives us more control over the state of the action by avoiding the possible instabilities and dependency issues that may come with latest versions"

Reply:
I'll make this change